services/QuillLMS/client/app/bundles/Staff/components/evidence/versionHistory/versionHistory.tsx

```diff
@@ -82,33 +85,40 @@ const VersionHistory = ({ history, match }) => {
   }
 
   const activityVersionDisplayValue = ({version}): string => {
-    if (version === 0) return 'Initial Version'
+    if (version === 1) return 'Initial Version'
```

Comment:
Other zero-index assumptions for this code exist:
1. https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/engines/evidence/app/controllers/evidence/feedback_controller.rb#L30
2. In the column migration. Create a new data migration that changes the default value to 1. 
3. Create a backfill script that moves all 0-valued records to 1-valued.

Alternatively, since this is rather annoying, you could push back on the spec. It's not clear to me that moving from 0-index to 1-index adds a lot of value in this case.